packages/flutter_tools/lib/src/fuchsia/fuchsia_sdk.dart

```diff
@@ -15,18 +23,56 @@ FuchsiaSdk get fuchsiaSdk => context[FuchsiaSdk];
 /// including a working fx command-line tool in the user's PATH.
```

Comment:
I wonder if this is a good place to have a comment keeping a list of the fx commands that are being used. The list could also be in the form of static const fields of FuchsiaSdk, e.g.:
static const _fxSyslogCommand = <String>["fx", "syslog"];

Reply:
Pulled all of the fx commands into static consts